src/common.h

```diff
@@ -1,3 +1,4 @@
+#pragma once
```

Comment:
This shouldn't be here. `#pragma`s are non-standard. There's an include guard directly underneath it, too.

Reply:
Isn't it ? I thought `#pragma once` was standard... or is it in C++ only ?

Reply:
It's nonstandard in both C and C++, as far as I know.